NuGet.Config

```diff
@@ -22,5 +22,6 @@
   </packageSources>
   <disabledPackageSources>
     <add key="buildtools" value="true" />
+    <add key="MyGetApplicationInsights" value="true" />
```

Comment:
This was part of your initial commit so I thought I will leave it there :-) Will remove